src/MasterPeace/Bundle/BookBundle/Entity/Question.php

```diff
@@ -93,9 +99,29 @@ public function setBook(Book $book)
     }
 
     /**
+     * @param Answer $answer
+     *
+     * @return Question
+     */
+    public function addAnswer(Answer $answer)
+    {
+        $this->answers[] = $answer;
+
+        return $this;
+    }
+
+    /**
+     * @param Answer $answer
+     */
+    public function removeAnswer(Answer $answer)
+    {
+        $this->answers->removeElement($answer);
```

Comment:
if ($this->answers->contains($answer)) {
            $this->answers->removeElement($answer);
        }

        return $this;